ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Overview/PlatformCvesOverviewPage.tsx

```diff
@@ -78,7 +85,25 @@ function PlatformCvesOverviewPage() {
         Cluster: data?.clusterCount ?? 0,
     };
 
-    const filterToolbar = <></>;
+    function onClearFilters() {
+        setSearchFilter({});
+        pagination.setPage(1, 'replace');
+    }
+
+    const filterToolbar = (
+        <AdvancedFiltersToolbar
```

Comment:
The generic `AdvancedFiltersToolbar` name seems beneficial to make the reference implementation in Vulnerabilities as easy as possible to imitate in other containers.

Reply:
Even considering that `includeCveSeverityFilters={false}` is a container-specific prop.

Reply:
+1  I wrote [a comment on the initial implementation](https://github.com/stackrox/stackrox/pull/11333/files#r1631599976) weighing the pros/cons of making this truly reusable at the global level versus at the `Container` level here. I opted for reusability at the latter for the reasons described above, but we could revisit this in the future if we think it is worthwhile to extend the scope.